README.md

```diff
@@ -145,6 +165,17 @@ class TestElement extends Polymer.Element {
   private _fooBarChanged(newFoo: number, newBar: string) {
     console.log(`foo is now: ${newFoo}, bar is now ${newBar}`);
   }
+  
+  // @listen can register both imperative or gesture listeners.  Your class must extend the
```

Comment:
You are correct, this can be better stated:

 ` @listen can register both gesture and non-gesture listeners declaratively.  To accomplish this, you must extend from both Polymer.GestureEventListeners and Polymer.DeclarativeEventListeners. 
`

Reply:
Take 3:   

````
  // @listen registers event listeners declaratively. 
  //
  // To use this decorator, your element class must extend from the supplied Polymer.DeclarativeEventListeners mixin.
  // Ex. class TestElement extends Polymer.DeclarativeEventListeners(Polymer.Element)
  //
  // The @listen decorator can also declaratively register gesture events. To accomplish this, 
  // you must extend from both Polymer.GestureEventListeners and Polymer.DeclarativeEventListeners. 
  //
  // Ex. class TestElement extends Polymer.DeclarativeEventListeners(Polymer.GestureEventListeners(Polymer.Element))
  //
  // listen(eventName: string, target: string|EventTarget)
  // @param eventName A string representing the event type to listen for
  // @param target A single element by id or EventTarget to target
  //

````